back-end/yunissaem_api/settings.py

```
@@ -0,0 +1,141 @@
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/5.2/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-(y=m@buaq-2z73de1d$%(8-(x$3&bjujblmz-93$3@6u6x*4)e'
```

Ruff (0.14.4)

Possible hardcoded password assigned to: "SECRET_KEY"

(S105)